slider/beatmap.py

```diff
@@ -1987,10 +1987,10 @@ def parse(cls, data):
                 milliseconds=_get_as_int(groups, 'General', 'AudioLeadIn', 0),
             ),
             preview_time=timedelta(
-                milliseconds=_get_as_int(groups, 'General', 'PreviewTime'),
+                milliseconds=_get_as_int(groups, 'General', 'PreviewTime', -1),
```

Comment:
Did you find this default somewhere in the docs, or why is this okay?

I don't think previous time really matters much, but jw

Reply:
it's in the linked docs: 
![image](https://user-images.githubusercontent.com/31628143/111923481-d4be9a80-8a75-11eb-81e4-8e3892c22141.png)

Reply:
apologies